### PR TITLE
Show ghost info in tile

### PR DESCRIPTION
**Pull request by bhollis**
This uses the d2ai info to add a label to ghosts:

Screen Shot 2019-08-02 at 11 21 19 PM

Note that all I used was location and "improved" (to add a plus for improved perks). I considered also mapping in an element, but I don't think anyone cares about telemetry.

This isn't localized yet. I suspect we'd want to tweak the localization even for English since not all the labels fit well.

### CODE REVIEW

**Comment by sundevour** [comment]
is the i18n intention to have these be brief possibly future-filter `^[a-z]$` strings or just as-short-as-reasonable planet representations?

**Comment by bhollis** [comment]
It's both so we can shorten them, and so other languages can use language-specific versions of the location.

**Comment by delphiactual** [comment]
we could use the planetary vendor icons...

**Comment by sundevour** [comment]
a bunch of those are just recolors without any meaning that i grasp

**Comment by delphiactual** [comment]
we could use the vendor bounty icons those are distinguishable easily

File: src/app/inventory/BadgeInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+      'item-equipment': !itemIs.bounty && !itemIs.stackable,
+      'item-stackable-max': itemIs.stackable && item.amount === item.maxStackSize,
+      'badge-capped': isCapped
+    };
+
+    const badgeContent =
+      (itemIs.bounty && `${Math.floor(100 * item.percentComplete)}%`) ||
+      (itemIs.stackable && (isCapped ? t('Badge.Max') : item.amount.toString())) ||
+      (itemIs.ghost && ghostBadgeContent(item)) ||
+      (!Object.values(itemIs).some(Boolean) && item.primStat && item.primStat.value.toString()) ||
```
Comment (bhollis) [inline src/app/inventory/BadgeInfo.tsx]: You can extract `Object.values(itemIs).some(Boolean)` into a local

File: src/app/inventory/BadgeInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+function ghostBadgeContent(item: DimItem) {
+  const infos = getGhostInfos(item);
+
+  const planet = _.uniq(infos.map((i) => i.location).filter((l) => l !== true && l !== false))
+    .map((location) => t(`Ghost.${location}`))
+    .join(',');
+  const improved = infos.some((i) => i.type.improved) ? '+' : '';
+
+  return [planet, improved];
```
Comment (bhollis) [inline src/app/inventory/BadgeInfo.tsx]: Whoa, never thought about returning an array of strings directly into JSX!

**Pull request merged**